media/js/products/relay/waitlist.es6.js

```diff
@@ -76,6 +76,14 @@ const WaitlistForm = {
                 newsletter: newsletter
             });
         }
+
```

Comment:
Note for reviewer: this file doesn't seem to be used anywhere (I think?), but I updated it anyway